src/generatedSheetFunctions.js

```diff
@@ -673,6 +673,29 @@ function SAN_VOLUME (projectSlug, from, to, interval = '1d') {
   )
 }
 
+
+/**
+* Returns the social volume for a slug.
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} source The source for which the social metric is fetching data. Example: "REDDIT"
+* @param {string} interval The resolution with which the data is fetched. Example: "5m"
+* @returns {Array} of the slug's social volume.
+* @customfunction
+*/
+function SAN_SOCIAL_VOLUME_NEW (projectSlug, from, to, source, interval = '1d') {
```

Comment:
Can't we replace SAN_SOCIAL_VOLUME?

Reply:
Yeah, you're right, removed it and just replaced the old function